### PR TITLE
feat: add `transaction` module

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/cell.ts [hunk 1]
```diff
@@ -6,6 +6,22 @@ export interface OutPoint {
   index: number
 }
 
+interface TypeScript {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/cell.ts]: interface Script is enough

File: packages/neuron-wallet/src/cell.ts [hunk 2]
```diff
@@ -6,6 +6,22 @@ export interface OutPoint {
...
+  version: number
+  args: string[]
+  signedArgs: string[]
+  reference: string | null
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/cell.ts]: reference: string | undefined, and better not to use null [guide](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined)

File: packages/neuron-wallet/src/cell.ts [hunk 3]
```diff
@@ -6,6 +6,22 @@ export interface OutPoint {
   index: number
 }
 
+interface TypeScript {
+  version: number
+  args: string[]
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/cell.ts]: in node we prefer to use Buffer or Uint8Array

File: packages/neuron-wallet/src/cell.ts [hunk 4]
```diff
@@ -6,6 +6,22 @@ export interface OutPoint {
...
+  signedArgs: string[]
+  reference: string | null
+  binary?: string
+}
+
+export interface Cell {
+  capacity: number
+  data: string
+  lock: string
+  type?: TypeScript | null
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/cell.ts]: use undefined

File: packages/neuron-wallet/src/transaction.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  inputs?: any
+  outputs?: Cell[]
+}
+
+// if addresses set to null, should be all transactions
+export const getTransactions = async (addresses: string[] | null, page: number, perPage: number) => {
+  const transaction: Transaction = mockedTransaction
+
+  return {
+    totalCount: page * perPage,
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/transaction.ts]: seems not good

File: packages/neuron-wallet/src/cell.ts (hunk 1 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/cell.ts]: OK

File: packages/neuron-wallet/src/cell.ts (hunk 2 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/cell.ts]: OK

File: packages/neuron-wallet/src/cell.ts (hunk 3 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/cell.ts]: Node use uint8array but RPC use hex string as input and output, I don't know why we should use uint8array

File: packages/neuron-wallet/src/transaction.ts (hunk 5 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/transaction.ts]: Just a mocked data.

File: packages/neuron-wallet/src/cell.ts (hunk 4 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/cell.ts]: typescript can be a script or null

File: packages/neuron-wallet/src/cell.ts (hunk 2 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/cell.ts]: should accept null as well, the RPC result will set to null, it should be compatible

File: packages/neuron-wallet/src/cell.ts (hunk 3 above)
Comment (ashchan) [reply packages/neuron-wallet/src/cell.ts]: I strongly recommend that we use `string` for CKB internal `[uint8]`, and `string[]` for `[[uint8]]` (`vec<vec<uint8>>`), as long as RPC returns hex string. Only use bytes when hashing internally.

**Pull request merged**